README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/expectedbehavior/acts_as_archival.svg?branch=master)](https://travis-ci.org/expectedbehavior/acts_as_archival)
```

Comment:
And yeah, I though maybe you had something in mind for rubocop (which maybe travis does now?).